### PR TITLE
Chat component: Add auto-scroll feature; Fix Lint & ESLint errors

### PR DESCRIPTION
**Pull request by thvphuc246**

### CODE REVIEW

**Review by voquanghien — COMMENTED** [review]
Please update chatbox as discussion yesterday @thvphuc246

File: src/app/_services/auth.service.ts [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import { Injectable } from "@angular/core";
 import { HttpClient, HttpHeaders } from "@angular/common/http";
 import { Observable } from "rxjs";
+import { User } from "../_models/user.model";
```
Comment (voquanghien) [inline src/app/_services/auth.service.ts]: Should write full url when import (ex: import A from "src/phase/containerA")
Please update for all

File: src/app/_services/chat.service.ts [hunk 2]
```diff
@@ -1,6 +1,5 @@
 import { Injectable } from "@angular/core";
 import { ChatMessage } from "../_models/chat-message.model";
-import { AuthService } from "../_services/auth.service";
```
Comment (voquanghien) [inline src/app/_services/chat.service.ts:3]: Same as above problem

File: src/app/message/message.component.scss [hunk 3]
```diff
@@ -1,18 +1,5 @@
 @import "variables", "hover", "shadows", "buttons", "functions", "gradients";
 
-.messageContainer {
-  display: flex;
```
Comment (voquanghien) [inline src/app/message/message.component.scss:4]: Use mixin for flex

File: src/app/_services/auth.service.ts (hunk 1 above)
Comment (thvphuc246) [reply src/app/_services/auth.service.ts]: > Should write full url when import (ex: import A from "src/phase/containerA")
> Please update for all

Should I use _**eslint-import-resolver-alias**_ for this case?

Comment (thvphuc246) [reply src/app/_services/auth.service.ts]: **_Update:_** It turns out no other plugins needed to be installed. I just need to define _compilerOptions.paths_ in _tsconfig.json_. But now VSCode sees my alias "@app" in "@app/_models/..." as a typo, though the project runs fine without errors. Tried adding this to .eslintrc but nothing changes:
```
"settings": {
        "import/reslover": {
            "node": {
                "paths": ["src"],
                "extensions": [".js", ".jsx", ".ts", ".tsx"]
            }
        }
    }
```

Comment (thvphuc246) [reply src/app/_services/auth.service.ts]: **_Update 2:_** Actually, it works now. Just need to restart my VSCode and the error is gone. The lines that I tried to add to _.eslintrc_ are not necessary either.

**Review by voquanghien — DISMISSED** [review]

**Review by voquanghien — COMMENTED** [review]
Please remove package-lock.json, we are using yarn not npm @thvphuc246

**Review by voquanghien — APPROVED** [review]

**Pull request merged**